torchsde/_core/methods/milstein.py

```diff
@@ -12,30 +12,67 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from ...settings import SDE_TYPES, NOISE_TYPES, LEVY_AREA_APPROXIMATIONS
+import math
+
+import torch
+
+from ...settings import SDE_TYPES, NOISE_TYPES, LEVY_AREA_APPROXIMATIONS, METHOD_OPTIONS as opt
 
 from .. import base_solver
 
 
-class Milstein(base_solver.BaseSDESolver):
+class BaseMilstein(base_solver.BaseSDESolver):
```

Comment:
Can `BaseMilstein` and `v_term` be made abstract?

What I mean is:

```
import abc

class BaseMilstein(base_solver.BaseSDESolver, metaclass=abc.ABCMeta):
    ...
    @abc.abstractmethod
    def v_term(self, I_k, dt):
        raise NotImplementedError
```

This signals that this class should never be instantiated itself, and that furthermore this method must be overriden by its subclasses.

Reply:
Sure, done!